adage/requirements.txt

```diff
@@ -1,9 +1,9 @@
 argparse==1.2.1
```

Comment:
From https://pypi.org/project/argparse/

> As of Python >= 2.7 and >= 3.2, the argparse module is maintained within the Python standard library. For users who still need to support Python < 2.7 or < 3.2, it is also provided as a separate package, which tries to stay compatible with the module in the standard library, but also supports older Python versions.

Should argparse be removed from environment?

Reply:
Yet another good point.